### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by traetox**
This reworks the kafka consumer to allow for dynamically deriving tags from kafka headers.

May of the other platforms that expect to send through kafka don't like to use lots of topics, so they send additional metadata via kafka message headers.

This will allow us to be more compatible with other kafka producers.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -20,10 +20,12 @@ require (
 	github.com/floren/o365 v0.0.1
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-ole/go-ole v1.2.4 // indirect
+	github.com/gobwas/glob v0.2.3
 	github.com/google/go-write v0.0.0-20181107114627-56629a6b2542
 	github.com/google/gopacket v1.1.17
 	github.com/google/uuid v1.1.1
 	github.com/gravwell/gcfg v1.2.5
+	github.com/gravwell/ingest/v3 v3.3.12
```
Comment (floren) [inline go.mod:28]: Can you run 'go mod tidy' and see if it gets rid of this? I don't see a reference to gravwell/ingest in your *code* so maybe it's just old stuff.

File: ingesters/kafka_consumer/main.go [hunk 2]
```diff
@@ -108,6 +108,9 @@ func main() {
 			}
 		}
 	}
+	for name, c := range cfg.Consumers {
+		fmt.Printf("%s: %+v\n", name, c)
```
Comment (floren) [inline ingesters/kafka_consumer/main.go:112]: Printf? Or should this use the logging stuff?

Comment (traetox) [reply ingesters/kafka_consumer/main.go]: resolved with https://github.com/gravwell/gravwell/pull/5/commits/f1e3846c2089a02ee247e0162af36eb5dcabc6f6

File: go.mod (hunk 1 above)
Comment (traetox) [reply go.mod]: go mod tidy appears to have cleaned all that garbage up.

**Comment by floren** [comment]
lgtm

**Pull request merged**